release-notes/1.20.0.md

```diff
@@ -0,0 +1,61 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.20.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* Anbox Cloud has introduced support for creating virtual machine instances. This is especially helpful for use cases that require a stronger isolation. You can use both containers and virtual machines for your applications.<!--AC-1459 and AC-2010 and AC-1916-->
+* The Anbox Management Service (AMS) snaps use [core22](https://snapcraft.io/core22).<!--AC-1853-->
+* AMS downloads the image from the image server only at first use instead of downloading all available images. <!--AC-1158-->
+* The watchdog can be enabled without specifying a boot package in the application manifest. <!--AC-2057-->
+* The Anbox Cloud images no longer start `android.hardware.media.c2` services automatically.<!--AC-2033-->
+* The included Mesa driver is tracking the latest 23.2.1 version and the `virglrenderer` is also tracking the latest upstream.<!--AC-1933-->
+* Some improvements to the swagger response for the AMS HTTP API are included. <!--AC-1923-->
+
+### Streaming improvements
+
+* The required NVIDIA driver version has been updated from the 525 series to 535 series.<!--AC-1924-->
+* Alpha quality support for Vulkan on NVIDIA GPUs is now available through a feature flag. To use this feature, you require newer NVIDIA drivers (versions 545 or later). Since the feature is at Alpha level, bugs and crashes can occur. If you are interested in early testing, contact us through the [users forum on discourse](https://discourse.ubuntu.com/c/anbox-cloud/users/148). Stable product support for this feature is planned for the 1.21 release.
```

Comment:
Experimental features for video degradation to support dynamic adaptive streaming in constrained-bandwidth situations in the [Anbox WebRTC streamer](https://anbox-cloud.io/docs/ref/webrtc-streamer). Below is an example of the streamer JSON file with this experimental feature enabled for 720p screen format:
```
{
  "video": {
    "experimental": {
      "degradation": {
        "level": 0.1,
        "texture_filter_mode": "linear"
      }
    },
    "bitrate_limits": [
      {
        "num_pixels": 921600,
        "min_kbps": 512,
        "max_kbps": 5000
      }
    ]
  }
}
```

Reply:
@keirthana as discussed, let's leave this out as it's still experimental. And if customers are interested in this feature, come and talk to us.